feature/src/main/java/com/eoyeongbooyeong/home/component/PlaceInfoBox.kt

```diff
@@ -0,0 +1,176 @@
+package com.eoyeongbooyeong.home.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import coil.compose.rememberAsyncImagePainter
+import coil.decode.SvgDecoder
+import coil.request.ImageRequest
+import com.eoyeongbooyeong.core.designsystem.theme.Black
+import com.eoyeongbooyeong.core.designsystem.theme.BooTheme
+import com.eoyeongbooyeong.core.designsystem.theme.Gray200
+import com.eoyeongbooyeong.core.designsystem.theme.Gray400
+import com.eoyeongbooyeong.core.designsystem.theme.White
+import com.eoyeongbooyeong.core.extension.noRippleClickable
+import com.eoyeongbooyeong.domain.Place
+import com.eoyeongbooyeong.feature.R
+
+@Composable
+internal fun PlaceInfoBox(
+    modifier: Modifier = Modifier,
+    place: Place,
+    onClick: () -> Unit = {},
+) {
```

Comment:
파라미터 선언시 필수가 먼저 나오고, 옵셔널이 그다음 나오는게 컴포즈 권장이에용! 그리고 옵셔널 중에서는 modifier가 제일 위로 와야 해요
```suggestion
internal fun PlaceInfoBox(
    place: Place,
    modifier: Modifier = Modifier,
    onClick: () -> Unit = {},
) {
```

Reply:
이 부분은 개인 성향 차이인 것 같지만, 저는 컴포넌트에 클래스 자체를 넘겨주는 것 보다는 원시타입을 보내는 것을 선호합니다.
해당 컴포넌트가 그 클래스에 너무 종속적이게 된다는 생각이 있어 필요한 요소들만 넘기는 것도 좋은 방법이라 생각해요